src/content/advanced/cluster-autoscaler/index.md

```diff
@@ -126,6 +126,19 @@ data:
       skipNodesWithLocalStorage: "true"
 ```
 
+### Balance similar node groups
+
+> Added in release v1.22.2-gs3
+
+The Cluster Autoscaler configuration by default doesn't differentiate between node groups when scaling. In case you want to enable this action, you need to set the following property to `"true"`.
```

Comment:
I would also link to https://docs.giantswarm.io/app-platform/app-configuration/ for details about how to submit the custom config

Reply:
It's already descibed on the page: https://docs.giantswarm.io/advanced/cluster-autoscaler/#how-to-set-configuration-options-using-the-user-values-configmap